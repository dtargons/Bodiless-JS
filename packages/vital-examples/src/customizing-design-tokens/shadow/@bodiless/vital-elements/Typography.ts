import { exampleTypography } from '../../../components/Typography';

export default exampleTypography;
