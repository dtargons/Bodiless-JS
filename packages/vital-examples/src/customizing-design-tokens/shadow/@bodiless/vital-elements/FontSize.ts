import { exampleTextDecoration } from '../../../components/TextDecoration';

export default exampleTextDecoration;
