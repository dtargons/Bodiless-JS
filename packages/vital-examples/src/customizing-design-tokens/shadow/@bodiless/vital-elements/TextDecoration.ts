import { exampleTextDecoration } from '../../../components/TextDecoration';
import { shadow } from '@bodiless/vital-elements';

export default shadow(exampleTextDecoration, 'Example:TextDecoration');
