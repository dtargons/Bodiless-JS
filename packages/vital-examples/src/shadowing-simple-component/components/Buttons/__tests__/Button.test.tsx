/* eslint-disable jest/valid-describe */
import { Button } from '@bodiless/fclasses';
import { testTokens } from '@bodiless/vital-elements';
import { exampleButtons } from '..';

describe('Buttons', testTokens(Button, exampleButtons));
