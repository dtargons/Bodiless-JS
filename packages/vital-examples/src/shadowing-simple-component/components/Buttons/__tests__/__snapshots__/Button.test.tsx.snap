// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Buttons Matches snapshot for Default 1`] = `
<span
  design={
    Object {
      "Body": [Function],
      "Wrapper": [Function],
    }
  }
  href="#"
/>
`;

exports[`Buttons Matches snapshot for Primary 1`] = `
<span
  design={
    Object {
      "Body": [Function],
      "Wrapper": [Function],
    }
  }
  href="#"
/>
`;

exports[`Buttons Matches snapshot for PrimarySelected 1`] = `
<span
  design={
    Object {
      "Body": [Function],
      "Wrapper": [Function],
    }
  }
  href="#"
/>
`;

exports[`Buttons Matches snapshot for Secondary 1`] = `
<span
  design={
    Object {
      "Body": [Function],
      "Wrapper": [Function],
    }
  }
  href="#"
/>
`;

exports[`Buttons Matches snapshot for SecondarySelected 1`] = `
<span
  design={
    Object {
      "Body": [Function],
      "Wrapper": [Function],
    }
  }
  href="#"
/>
`;

exports[`Buttons Matches snapshot for WhereToBuy 1`] = `
<span
  design={
    Object {
      "Body": [Function],
      "Icon": [Function],
      "Wrapper": [Function],
    }
  }
>
  Where to Buy
</span>
`;

exports[`Buttons Matches snapshot for WhereToBuyWithoutIcon 1`] = `
<span
  design={
    Object {
      "Body": [Function],
      "Icon": [Function],
      "Wrapper": [Function],
    }
  }
>
  Where to Buy
</span>
`;

exports[`Buttons Matches snapshot for WithArrow 1`] = `
<span
  design={
    Object {
      "Icon": [Function],
      "Wrapper": [Function],
    }
  }
/>
`;

exports[`Buttons Matches snapshot for WithBigButton 1`] = `
<span
  design={
    Object {
      "Body": [Function],
      "Wrapper": [Function],
    }
  }
  href="#"
/>
`;

exports[`Buttons Matches snapshot for WithDisabled 1`] = `
<span
  design={
    Object {
      "Wrapper": [Function],
    }
  }
/>
`;

exports[`Buttons Matches snapshot for WithPrimary 1`] = `
<span
  design={
    Object {
      "Body": [Function],
      "Wrapper": [Function],
    }
  }
  href="#"
/>
`;
